cubano-concordion/src/main/java/org/concordion/cubano/driver/concordion/HttpEasyDownloader.java

```diff
@@ -1,24 +1,27 @@
-package org.concordion.cubano.driver.http;
-
-import java.io.File;
-import java.io.IOException;
-
-/**
- * File downloader.
- */
-public class HttpEasyDownloader implements HttpDownloader {
-
-    /**
-     * Download file.
-     */
-    public File downloadFile(String url, String outputFolder, String cookieHeader) throws IOException {
-        return HttpEasy.request().
-                header("Accept-Language", "en-US,en;q=0.8").
-                header("User-Agent", "Java").
-                header("Referer", "google.com").
-                header("Cookie", cookieHeader).
-                path(url).
-                get().
-                downloadFile(outputFolder);	
-    }
+package org.concordion.cubano.driver.concordion;
```

Comment:
I've moved pagegrabber package next to HtmlCaptureExtension as this exists only to support that. 
I'm about to raise another PR (new PR should be done first if decide to go with it) that removes HttpDownloader entirely, again because it only existed to support CaptureExtension